TrajectoryPlannerROS:
  max_vel_x: 0.4
  min_vel_x: 0.1
  max_vel_theta: 1.0
  min_in_place_vel_theta: 0.4
  acc_lim_theta: 2.0
  acc_lim_x: 2.5
  acc_lim_y: 0.0

  yaw_goal_tolerance: 0.1
  xy_goal_tolerance: 0.2
  latch_xy_goal_tolerance: true

  sim_time: 1.7
  vx_samples: 6
  vtheta_samples: 20
